Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -2842,75 +2817,68 @@ public QueryResult<BaseItem> GetItems(InternalItemsQuery query)
                 statementTexts.Add(commandText);
             }
 
-            using (WriteLock.Read())
+            using (var connection = GetConnection(true))
             {
-                using (var connection = CreateConnection(true))
+                var statements = PrepareAllSafe(connection, statementTexts);
+
+                if (!isReturningZeroItems)
                 {
-                    return connection.RunInTransaction(db =>
```

Comment:
I'm wary that if you run without locking and without transactions, then those two queries could potentially return different results if DB was changed between those...

Reply:
More likely, SQLite will return a LOCKED notification back to the reader which will result in an exception..  Using transaction for SELECT (READS) is not required and eventually results in 'transaction in transaction' errors.